diplomacy/web/amplify/backend/function/shadefn1f4a9/src/index.js

```diff
@@ -0,0 +1,113 @@
+/* Amplify Params - DO NOT EDIT
+	AUTH_SHADE58086F6F58086F6F_USERPOOLID
+	ENV
+	REGION
+Amplify Params - DO NOT EDIT */
+
+/*
+ * This file handles accepting an API reques to consent/ and, if the session is
+ * valid, saves the user consent (terms of use) acceptance as a user attribute on
+ * the cognito user pool.
+ */
+
+const AWS = require('aws-sdk');
```

Comment:
This is the only backend file- the handler for `/consent` to update the user accepted consent at date within the cognito user pool.